SparkPost/matchbox
# FE-819 Update Page Component

### What Changed
- Restyles the Page component
- Does not touch Popover, EmptyState, or Actionlist, which are all dependencies of Page
- Removes the default prop `empty: {}`

<img width="998" alt="Screen Shot 2020-03-19 at 3 16 43 PM" src="https://user-images.githubusercontent.com/3903325/77106030-e13c8d00-69f4-11ea-9d98-05552ba15240.png">

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit and verify Page stories

### PR Checklist
